core/chaincode/exectransaction.go

```diff
@@ -139,6 +140,9 @@ func ExecuteTransactions(ctxt context.Context, cname ChainName, xacts []*pb.Tran
 	txerrs = make([]error, len(xacts))
 	for i, t := range xacts {
 		_, txerrs[i] = Execute(ctxt, chain, t)
+		if txerrs[i] != nil {
+			sendTxRejectedEvent(xacts[i].Uuid, txerrs[i].Error())
```

Comment:
@tamasblummer what is your opinion about this idea? We agreed that the event should be sent after the chaincode fails.